test/library/standard/List/perf/listBenchmark1.chpl

```diff
@@ -0,0 +1,182 @@
+//
+// A performance test benchmarking common list operations.
+//
+
+private use HaltWrappers;
+private use List;
+private use Random;
+private use Time;
+
+type byte = int(8);
+type testList = list(byte, false);
+
+config param isPerformanceTest: bool = false;
+config const n0: int = 50000;
+
+const n1: int = n0 / 1000;
+const seed = 314159;
+
+proc createList(size: int) {
+  var result: testList;
+  for i in 1..size do result.append(0:int(8));
+  return result;
+}
+
+proc setup() {
+  extern proc srand(seed: int);
+  srand(seed);
+}
+
+proc randInt() {
+  extern proc rand(): int;
```

Comment:
Why not use the Random module here?

Reply:
Ah. I just copied what the old `timeVectorArray.chpl` test did. I will look into using Random instead.